source/tutorials/module-system/a-basic-module/default.nix

```
@@ -1,10 +1,10 @@
-{ pkgs ? import <nixpkgs> { } }:
 let
+  pkgs = import <nixpkgs> {};
   result = pkgs.lib.evalModules {
     modules = [
       ./options.nix
       ./config.nix
     ];
   };
 in
-result.config
+  result.config
```

nit: leave the white space before `result.config` out. Which formatter are you using? `nixfmt-rfc-style` was made official back in March I believe.

```suggestion
result.config
```